packages/create-cloudflare/src/frameworks/versionMap.json

```
@@ -1,5 +1,5 @@
 {
-	"angular": "next",
+	"angular": "latest",
```

Sorry for the churn but please can we change this to

```suggestion
	"angular": "16.0.x",
```